test_common/harness/conversions.h

```diff
@@ -119,6 +119,133 @@ static inline int IsHalfSubnormal( cl_half x )
     return ( ( x & 0x7fffU ) - 1U ) < 0x03ffU; 
 }
 
+static inline float half2float(cl_ushort us)
```

Comment:
since we're moving these, it might be a good time to change them to use cl_half throughout.

Reply:
Sorry for confusion. This PR has to be closed. This is the old version of code. The new https://github.com/KhronosGroup/OpenCL-CTS/pull/1074 do not uses this function and uses from OpenCL headers  https://github.com/KhronosGroup/OpenCL-Headers/blob/master/CL/cl_half.h#L361